### PR TITLE
[Merged by Bors] - feat(SetTheory/Cardinal/Basic): there exists a well-founded linear ordering of any type

### PR DESCRIPTION
**Pull request by vihdzp**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by j-loreaux — COMMENTED** [review]
Thanks!

bors merge

File: Mathlib/SetTheory/Cardinal/Basic.lean [hunk 1]
```diff
@@ -971,6 +971,11 @@ instance WellOrderingRel.isWellOrder : IsWellOrder α WellOrderingRel :=
 instance IsWellOrder.subtype_nonempty : Nonempty { r // IsWellOrder α r } :=
   ⟨⟨WellOrderingRel, inferInstance⟩⟩
 
+variable (α) in
+theorem exists_wellOrder : ∃ (_ : LinearOrder α), WellFoundedLT α := by
```
Comment (j-loreaux) [inline Mathlib/SetTheory/Cardinal/Basic.lean:975]:
```suggestion
variable (α) in
/-- The **well-ordering theorem** (or **Zermelo's theorem**): every type has a well-order -/
theorem exists_wellOrder : ∃ (_ : LinearOrder α), WellFoundedLT α := by
```

**Review by j-loreaux — COMMENTED** [review]
bors d+

File: Mathlib/SetTheory/Cardinal/Basic.lean (hunk 1 above)
Comment (vihdzp) [reply Mathlib/SetTheory/Cardinal/Basic.lean:975]: I'm a bit surprised no other theorem had that docstring already.

**Pull request closed without merging**